examples/transfer/plot_rpa_steps.py

```diff
@@ -0,0 +1,111 @@
+"""
+====================================================================
+Plot the data transformations in the Riemannian Procrustes Analysis
+====================================================================
+
+Use the SpectralEmbedding module to plot in 2D the transformations on the data
+points from source and target domains when applying the Riemannian Procrustes
+Analysis [1]_ to match their statistics.
+
+    .. [1] `Riemannian Procrustes analysis: transfer learning for
+    brain-computer interfaces
+    <https://hal.archives-ouvertes.fr/hal-01971856>`_
+    PLC Rodrigues et al, IEEE Transactions on Biomedical Engineering, vol. 66,
+    no. 8, pp. 2390-2401, December, 2018
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+
+from pyriemann.embedding import SpectralEmbedding
+from pyriemann.datasets.simulated import make_example_transfer_learning
+
+from pyriemann.transfer import (
+    decode_domains,
+    TLCenter,
+    TLRotate,
+)
+
+
+# fix seed for reproducible results
+seed = 66
+
+# create source and target datasets
+n_matrices = 50
+X_enc, y_enc = make_example_transfer_learning(
```

Comment:
```suggestion
X_enc, y_enc = make_classification_transfer(
```

I would prefer to avoid _example and it's information to say it's for classification.